_includes/registration-form.html

```diff
@@ -133,6 +134,18 @@
                         placeholder="Sie können uns hier einen Kommentar hinterlassen" name="kommentar" id="kommentar" class="form-control" rows="3"></textarea>
             </fieldset>
 
+            <fieldset>
+                <legend><small>Datenschutz</small></legend>
+                <input type="checkbox" id="disclamer"
+                    name="disclamer"
+                    value="ja"
+                    class="form-check-input ml-1"
+                    required="required"
+                />
+                <label for="disclamer" class="form-text text-muted ml-4 small">* Ich bin einverstanden, dass meine personenbezogenen Daten erhoben, gespeichert und verarbeitet werden. (Buchhaltung, Anwesenheitslisten, Teilnahmebescheinigungen u.ä.)</label>
+                <small>Alle personenbezogenen Daten werden entsprechend den jeweils geltenden Vorschriften zum Schutz personenbezogener Daten, nur zum Zwecke der Tagungsorganisation und Abwicklung erhoben, verarbeitet und genutzt. Eine Weitergabe an Dritte erfolgt nicht.</small>
```

Comment:
Das Komma würde ich entfernen
```suggestion
                <small>Alle personenbezogenen Daten werden entsprechend den jeweils geltenden Vorschriften zum Schutz personenbezogener Daten nur zum Zwecke der Tagungsorganisation und Abwicklung erhoben, verarbeitet und genutzt. Eine Weitergabe an Dritte erfolgt nicht.</small>
```